osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/config/ThrottlingClientConfig.java

```diff
@@ -10,15 +10,22 @@
 package org.opensmartgridplatform.adapter.protocol.dlms.application.config;
 
 import java.time.Duration;
+import lombok.extern.slf4j.Slf4j;
 import org.opensmartgridplatform.throttling.ThrottlingClient;
+import org.springframework.beans.factory.BeanInitializationException;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Conditional;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.web.client.ResourceAccessException;
 
+@Slf4j
 @Configuration
 public class ThrottlingClientConfig {
 
+  private static final int REGISTER_RETRY_DELAY = 5000;
```

Comment:
Why not something like:
```java
  @Value("#{T(java.time.Duration).parse('${throttling.register.retry.delay:PT5S}')}")
  private Duration registerRetryDelay;
```
It does not need to be added to the properties per se, since the default will do, at least for now, but it enables overriding without code changes, if some other value may turn out to be more desirable.
It would look more consistent with all things being configurable in this class, but it may be better if you changed this, to rename the current `delay` (perhaps something like `permitRejectedDelay`) to set the two delays apart.

Reply:
Changed both properties and added it to properties file. I prefer all configurable properties to be in the default properties file